src/flakeref.rs

```diff
@@ -313,7 +313,7 @@ impl Display for FlakeRefType {
                 write!(f, "{uri}")
             }
             FlakeRefType::GitForge {
-                src,
+                platform: src,
```

Comment:
```suggestion
                platform,
```
And in the macro as well.

Reply:
dammit: never trust an LSP change, I guess.

Reply:
It probably can't change in the format macro yet, so it tries to stay semantically correct.